awot/graph/sonde.py

```diff
@@ -6,7 +6,7 @@
 from .skew import SkewXTick
 
 
-def plot_skewt_logp(data, **kwargs):
+def plot_skewt_logp(data, instance, **kwargs):
 
     '''
     method to plot sounding or dropsonde data

```

Comment:
We'll need some docstring info to help in useage. See other IO routines for examples.